Modules/Product/Http/Services/ProductService.php

```diff
@@ -0,0 +1,1184 @@
+<?php
+
+namespace Modules\Product\Http\Services;
+
+use Modules\Product\Entities\Product;
+use Illuminate\Support\Str;
+use Modules\Product\Entities\Video;
+use Modules\Product\Entities\ProductVariation;
+use Modules\Product\Entities\ProductImage;
+use Modules\Product\Entities\ProductPrepack;
+use Modules\Product\Entities\Category;
+use File;
+use DB;
```

Comment:
I think we don't need this

Reply:
Yes we are using it.  Please have a look sample 
$productPrepack = new ProductPrepack(); 
Line 422, File [ProductService.php]

Reply:
those are old versions, for new version we are using
Illuminate\Support\Facades\DB;